mne/time_frequency/tfr.py

```diff
@@ -2409,9 +2409,11 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     if copy is None:
         copy = baseline is not None
     data = rescale(data, times, baseline, mode, copy=copy)
+    coef = 10
 
     if np.iscomplexobj(data):
         data = np.sqrt((data * data.conj()).real)
+        coef = 20  # because of sqrt, data is amplitude(-like) not power
```

Comment:
... but really, if this is the only place we do this, then we should just do:
```suggestion
        data = (data * data.conj()).real
```
By avoiding the sqrt you avoid the need to change the coef at all (the sqrt == raising to 1/2 pow == factor of 2 mult after dB conv)

But we'll see how it shakes out after you fix the `np.iscomplexobj` stuff...

Reply:
I wondered about removing the `sqrt`...  but that would change the behavior for the `dB=False` case. Is that OK?

Reply:
It seems like it will change, but probably in a good way in the sense that it's more consistent. It seems like if you plot power or complex they should look the same in dB mode and in non-dB mode. We should just clarify that what we plot is power in both cases (rather than amplitude in one and power in the other, when in non-dB mode)

Reply:
ok great, just confirming that there wasn't a compelling reason that it should stay as amplitude in the "complex & non-dB" case